metadata-jobs/mae-consumer-job/src/main/java/com/linkedin/metadata/kafka/elasticsearch/MCEElasticEvent.java

```diff
@@ -1,4 +1,4 @@
-package com.linkedin.metadata.utils.elasticsearch;
+package com.linkedin.metadata.kafka.elasticsearch;
```

Comment:
Is this class getting referenced anywhere? Why do we need an MCE related class in the MAE consumer?

Reply:
I agree it seems misnamed, but it looks used / needed.

`ElasticEvent` (the interface) is referenced in more places. This is just a concrete implementation (the only one?), and is created here.

https://github.com/linkedin/datahub/blob/f103998dcb1ade1a0b21fce4b1cbcb6bb17a37e9/metadata-jobs/mae-consumer-job/src/main/java/com/linkedin/metadata/kafka/MetadataAuditEventsProcessor.java#L112

`ElaticEvent` looks like it contains basic information about an update, like which index it should go to, and the id of the document. The `buildJson` method is abstract. So the MAE job will take the MAE, pull off the aspects, transform them to search documents, and create an `ElaticEvent` for each that determines the ID and index to send the document to.

Reply:
Got it. Should we rename it to MAEElasticEvent?

Reply:
Good ideas. I would suggest doing that refactoring as a separate PR to keep this one contained.